doc/source/ScriptingAPI/ScriptFiles/index.md

```diff
@@ -0,0 +1,82 @@
+# Script Files #
+
+Gaffer script files – the projects you load and save – are simply a sequence of Python code that carry all the instructions necessary to reconstruct a node graph. When you save a script in Gaffer, the serializer engine takes the nodes and plugs of the current graph and writes them in sequence to an unbinarized file with a Gaffer extension in UTF-8 encoding.
+
+> Important :
+> Scripts loaded into the application and code copy-pasted from or to the _Graph Editor_ use the same syntax. They are safely interchangeable.
+
+You can copy a part or all of a node graph in the _Graph Editor_ (select the nodes, then hit <kbd>Control</kbd> + <kbd>C</kbd>) and paste it into the _Script Editor_ or an external text editor. If you find yourself struggling to find a way of constructing a portion of a graph using script, you can open a sample script in an external text editor and examine how it is constructed.
+
+
+<!-- TODO: Find a more precise URL -->
+<!-- TODO: Make a proper list of places -->
+<!-- > Tip : Sample Gaffer scripts can be found in the [source code itself](https://github.com/GafferHQ/gaffer/tree/0.47.0.0). -->
```

Comment:
I didn't think this was a very helpful pointer. Once the `/resources` folder has more scripts from tutorials, this should point there.